docs/embedding/introduction.md

```diff
@@ -1,165 +1,39 @@
 ---
-title: Embedding Metabase in other applications
+title: Introduction
 redirect_from:
   - /docs/latest/administration-guide/13-embedding
-  - /docs/latest/administration-guide/12-public-links
 ---
 
-# Embedding Metabase in other applications
+# Introduction
 
-Metabase includes a powerful application embedding feature that allows you to embed your saved questions or dashboards in your own web applications. You can even pass parameters to these embeds to customize them for different users.
+Embedding allows people to view your Metabase app inside another web app (such as your company's website).
```

Comment:
```suggestion
You can embed Metabase tables, charts, and dashboards—even Metabase's query builder—in your website or application.
```

This phrasing makes it a little more concrete, by saying explicitly what you can embed